index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/8496e48197.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Luis Portal</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="toolbar-dark-compact">
+            <div class="logo-placeholder"><img src="assets/images/my_logo.png" alt="Portal logo"></div>
+            <div class="normal-button"><i class="fa-solid fa-bars"></i></div>
+        </header>
+
+        <section class="main-content">
+            <h1>I’m Mariane Glad to see you!</h1>
+            <h5>I’m a software developer! I can help you build a product , feature or website Look through some of my
```

Comment:
- [ ] This paragraph should be a `p` tag. This way will be semantically correct and the screen readers and other technologies will thank you. 👍

Reply:
It's done.